mazegame/Assets/Scripts/Maze/TextEditor.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TextEditor : MonoBehaviour
+{
+
+    string key1_found = "You found a key! I wonder where it fits...";
+    string key2_found = "You found a second key!";
+    string key3_found = "You found a third key! ... *click*... Sounds like a door has opened";
```

Comment:
From what i can tell, these class member variables are never used anywhere else apart from in one instance inside the class. 
1. Move these to `Start` function
2. Make them type `var`
3. Avoid the use of numbers in variable names. Please rename to `first_key_found`